.travis.yml

```diff
@@ -1,20 +1,28 @@
 dist: xenial
 language: python
 python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
-  - "2.7"
+  - '3.5'
```

Comment:
please avoid making changes in this PR that are unrelated to the feature you are adding.

Reply:
Sorry.